modules/ROOT/pages/functions/scalar.adoc

```diff
@@ -1427,3 +1427,70 @@ The relationship type of `r` is returned.
 
 ======
 
+[[functions-valueType]]
+== valueType()
+
+The function `valueType()` returns a `STRING` representation of the most precise value type that the given expression evaluates to.
+
+*Syntax:*
+
+[source, syntax, role="noheader"]
+----
+valueType(expression)
+----
+
+*Returns:*
+
+|===
+
+| `STRING`
+
+|===
+
+*Arguments:*
+
+[options="header"]
+|===
+| Name | Description
+
+| `expression` | Any expression that returns a value.
+
+|===
+
+*Considerations:*
+
+It is possible that future releases of Cypher will include updates to the current type system.
+This makes it possible for more precise types to be added which would be returned instead.
+As this would be a breaking change to this function, they will not be returned until the following major release.
+For example, the function currently returns `"FLOAT"`, but if a more specific `FLOAT` type was added, e.g. `FLOAT32` this would be considered more specific and not returned until the next major release, and `"FLOAT"` would continue to be returned for any `FLOAT32` values.
+Completely new types may still be added in minor versions.
+This should be taken into account when relying on the output of the `valueType() function.
+See the xref::syntax/expressions.adoc#type-predicate-expressions[type predicate expression] for an alternative way of testing type values.
```

Comment:
Seems like we may want to be more specific with which types are currently supported.

I guess the easiest would be to add the list of supported types here so it doesn't reference another place?

cc @JPryce-Aklundh penny for your thoughts? :)